template/plop/reactComponent/index.js

```diff
@@ -4,6 +4,10 @@ const templatesPath = path.join(__dirname, 'templates');
 
 module.exports = (plop) => {
   const containerDirectory = 'src/{{ directory }}/{{ camelCase name }}';
+
+  const projectPathAbsolute = plop.getDestBasePath();
+  const containerDirectoryAbsolute = path.join(projectPathAbsolute, containerDirectory);
```

Comment:
Shouldn't this say `componentDirectoryAbsolute`? (Also `containerDirectory` -> `componentDirectory`)